src/main/java/org/jenkinsci/plugins/casc/integrations/GroupPermissionDefinition.java

```diff
@@ -0,0 +1,47 @@
+package org.jenkinsci.plugins.casc.integrations;
+
+import hudson.security.Permission;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.util.*;
+
+/**
+ * Created by mads on 2/7/18.
+ */
+public class GroupPermissionDefinition {
+
+    private String name;
+    private Collection<String> permissions = new ArrayList<>();
```

Comment:
initializer is unused from what I see

Reply:
done. Removed the intialized, made the intialization in constructor.